types/jvm_types.pp

```diff
@@ -0,0 +1 @@
+type Jira::Jvm_types = Enum['openjdk-11', 'oracle-jdk-1.8', 'custom']
```

Comment:
can you add the puppet-strings style `# @summary ...` comment?

Reply:
Done.